gradle/libs.versions.toml

```diff
@@ -5,7 +5,7 @@ android-minSdk = "21"
 android-targetSdk = "34"
 compose = "1.6.7"
 java = "1.8"
-kotlin = "2.0.0"
+kotlin = "1.9.23"
```

Comment:
What type of breaking changes do you mean? I don't think there should be any, except of course the increase in minimal Kotlin version for library consumers. That increase either happens now or in a future major. 

(But I haven't seen it compile yet with 2.0.0, so it's not really an option I think.)

Reply:
Yeah mostly that's what I mean, if we update to 2.0 in the future, it might be a breaking change for consumers... But as you said, since this is not working, we can consider that in the future when we bump the version yeah.